src/cluster_legacy.c

```diff
@@ -4872,6 +4872,8 @@ static int clusterNodeCronHandleReconnect(clusterNode *node, mstime_t handshake_
     /* A Node in HANDSHAKE state has a limited lifespan equal to the
      * configured node timeout. */
     if (nodeInHandshake(node) && now - node->ctime > handshake_timeout) {
+        serverLog(LL_WARNING, "Clusterbus handshake timeout [%s]:%d after a timeout of %lld", node->ip,
+                  node->cport, handshake_timeout);
```

Comment:
```suggestion
        serverLog(LL_WARNING, "Clusterbus handshake timeout %s:%d after %lldms", node->ip,
                  node->cport, handshake_timeout);
```
Minor suggestion to indicate clearly the timeout is in ms and remove the square bracket. There is some usage of both with and without the square bracket.

Reply:
Complete